.github/workflows/net-8-native-aot-pipeline-canary.yaml

```diff
@@ -13,26 +13,26 @@ jobs:
       steps:
         - uses: actions/checkout@v2  
         - name: Build and Deploy
-          uses: ./.github/actions/sam-build-and-deploy
+          uses: ./.github/actions/sam-build-and-deploy-native
           with:
             aws-access-key: ${{ secrets.AWS_ACCESS_KEY_ID }}
             aws-secret-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
             aws-region: ${{ secrets.AWS_REGION }}
             dotnet-version: '8.x'
-            template-file-path: ./src/NET8/template.yaml
-            project-directory: ./src/NET8/
```

Comment:
Quick check: is `./src/NET8` still used?  Or is it fully replaced by `./src/NET8Native/`?

Reply:
If `./src/NET8` is no longer needed, can it be removed?

Reply:
NET8 is still used for the non-native .NET 8 canary

Reply:
This was just a copy/paste miss when creating the native canary from the non-native canary

Reply:
It's used here: https://github.com/aws-samples/serverless-dotnet-demo/blob/main/.github/workflows/net-8-pipeline-canary.yaml

Reply:
Sounds good - just wanted to confirm.